### DISCUSSION TITLE
Getting kicked while on a VPS

### DISCUSSION THREAD

**Discussion by HysteriaMC** [ORelio/Minecraft-Console-Client#2030] [Q&A]
Hey,

So I have my MCC setup on a VPS, and it randomly disconnects from the server every now and again? Probably about twice a day, the server doesn't restart and there doesn't seem to be any rhyme or reason for it. Does anyone know why please?

Thanks!

**Comment by Daenges** [comment]
Yu could use [AutoRelog](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/Resources/config/MinecraftClient.ini#L130).

**Comment by rozatoo** [comment]
This might be happening because of a spike in CPU usage MCC sometimes has. Check if your CPU usage goes up higher than it should moments before the disconnect
